deepspeed/runtime/progressive_layer_drop.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+from deepspeed.runtime.constants import PLD_THETA, PLD_GAMMA
+from deepspeed.utils import log_dist
+
+
+class ProgressiveLayerDrop(object):
+    def __init__(self, pld_params):

```

Comment:
The parameters are dynamically updated in each forward pass. We put them into a dict so that they can be passed to the DeepSpeed forward pass using kargs, without changing the API.

Reply:
Done